src/app/demo/demo.component.ts

```diff
@@ -26,4 +33,9 @@ export class DemoComponent implements OnInit {
 
   ngOnInit() {
   }
+
+  changeDirTo(dir: langDir) {
```

Comment:
Can you please add a short comment to the method that `dir` is about right-to-left (`rtl`) and left-to-right (`ltr`) and therefore the method allows changing the screen orientation (?) of the demo. I think this helps people understand what is happening here

Reply:
sure.